README.md

```diff
@@ -102,6 +102,32 @@ If you only need to add a new Spryk configuration you will start by adding an In
 
 When this is done run the Integration tests with `vendor/bin/codecept run Integration -g {YOUR TEST GROUP}` and see the test failing. You will get a message that the Spryk definition was not found by the given name, so add the definition file for you new Spryk. 
 
-You need to add your Spryk definition file into `vendor/spryker-sdk/spryk/config/spryk/spryks` when you added your new Spryk definition file re-run the tests. Now it dependends on the spryk type you choosed for your Spryk definition what comes next.
+You need to add your Spryk definition file into `config/spryk/spryks` on project or core layer:
```

Comment:
layer or level?
I think we usually refer to the subnames as (application or otherwise) layers.